app/models/course/lesson_plan/item.rb

```diff
@@ -38,4 +40,9 @@ def validate_presence_of_bonus_end_at
       errors.add(:bonus_end_at, :required)
     end
   end
+
+  def start_at_cannot_be_after_end_at
```

Comment:
For convention, we usually start our validation methods with `validate`. Suggest you rename to `validate_start_at_before_end_at`.